eth/stagedsync/stage_execute.go

```diff
@@ -130,12 +130,17 @@ func executeBlock(
 	_, isPoSa := cfg.engine.(consensus.PoSA)
 	getHashFn := core.GetHashFn(block.Header(), getHeader)
 	if isPoSa {
-		receipts, err = core.ExecuteBlockEphemerallyForBSC(cfg.chainConfig, &vmConfig, getHashFn, cfg.engine, block, stateReader, stateWriter, epochReader{tx: tx}, chainReader{config: cfg.chainConfig, tx: tx, blockReader: cfg.blockReader}, contractHasTEVM)
+		getTracer := func(txIndex int, txHash ecom.Hash) (vm.Tracer, error) {
```

Comment:
check if you can use _ rather than using argument names here (txIndex,txHash) since they are not used

Reply:
I think there is no way